src/app_requirements.txt

```diff
@@ -22,7 +22,7 @@ newrelic==4.20.1.121
 openapi-spec-validator==0.2.7
 pathlib==1.0.1
 psutil==5.6.3
-pynamodb==3.4.1
```

Comment:
This and the 2 above package version upgrades were required for the index and query support.